### ISSUE TITLE
Loss NAN

### ISSUE THREAD

**Issue by TPZZZ** [codeslake/IFAN#16]
hello Junyong Lee:
Thanks to your great work!
I got a problem of Loss nan when I retrained the IFAN using my training setting, without using the losses reblurring and defocus map estimation by duai pixel data.
Do these losses make the loss be nan during the training?
Thank you for your help! @codeslake

**Comment by codeslake** [comment]
You can try to decrease your learning rate or [gradient clipping value](https://github.com/codeslake/IFAN/blob/main/configs/config.py#L43).
